docs/modules/layers.rst

```diff
@@ -49,6 +49,8 @@ Layer classes: convolutional layers
 .. autoclass:: Conv2DLayer
     :members:
 
+.. autofunction:: lasagne.layers.conv.conv_output_length
+
```

Comment:
I guess I'd keep that one private, so users browsing the reference don't get confused. When implementing their own convolutional layer, they will probably use an existing implementation as the basis and see the helper function anyway. But I don't have a strong opinion on that, do as you please!